### ISSUE TITLE
Make PDFs into static webpages (D&I report, Impact Report, one-sheet)

### ISSUE THREAD

**Issue by allyceh** [usds/website#704]
Source: USDS.gov redesign questions & wishlist items in Slack

"Make any PDFs into static webpages instead (D&I report, Impact Report, one-sheet) - you could keep the option to download the PDF but yes, bring the content into HTML"

- [ ] We will want to test this page for accessibility once it's available in the staging environment.

**Comment by bendelphinegov** [comment]
the two-column design of the draft D&I report looks like a pretty great template as-is (as it can just skip to one column on mobile). would prob just want some sort of sub-nav or pagination to deal with the length on mobile

@kategreenUSDS for speed, it might be easiest to build this out as a raw HTML file and skip the markdownification step, though I don't know how that affects other aspects of deployment
 
(note that I'm purposefully not sharing the pdf here as it's still in draft/dontshare mode)

**Comment by kategreenUSDS** [comment]
this is on hold inside OMB, so we're going to close this out for now.
